BASE/Test/Microsoft.ApplicationInsights.Test/ApplicationInsightsTypes/ApplicationInsightsTypes.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Test.props'))\Test.props" />
```

Comment:
Do we plan to extract this to a common place? (or merged with `$(PropsRoot)\Test.props`?)

Reply:
I'm slowly migrating all of these legacy props to the newer PropsRoot.
My goal is to remove the legacy ones and have all projects using the common props